src/pageEditor/sidebar/ActivatedModComponentListItem.tsx

```diff
@@ -55,37 +55,37 @@ import { type ModComponentBase } from "@/types/modComponentTypes";
  * @see DynamicModComponentListItem
  */
 const ActivatedModComponentListItem: React.FunctionComponent<{
-  extension: ModComponentBase;
-  recipes: ModDefinition[];
+  modComponentBase: ModComponentBase;
```

Comment:
NIT: No need to call this `modComponentBase`.
```suggestion
  modComponent: ModComponentBase;
```